uk.co.compucorp.civicrm.tasksassignments/CRM/Activity/Service/ActivityService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class CRM_Activity_Service_ActivityService {
+
+  /**
+   * Returns the activity types for a certain component. Default limit is 25
+   *
+   * @param $componentName
+   *  The name of the component
+   * @param int $limit
+   *  The maximium number of results to return
+   *
+   * @return array
+   */
+  public static function findByComponent($componentName, $limit = 0) {
```

Comment:
I think this would make the default 0, right?  So either update doc-block or the default value for $limit.